src/service/url-replacements-impl.js

```diff
@@ -513,6 +514,21 @@ export class GlobalVariableSource extends VariableSource {
       return this.getStoryValue_(storyVariables => storyVariables.pageId,
           'STORY_PAGE_ID');
     });
+
+    this.set('CONTAINER_TOKEN', () => {
```

Comment:
what is the use case of this?
we can probably make this a macro function that takes an argument, like we have for `QUERY_PARAM('ct')`, we can have `FRAGMENT_PARAM('ct')`